tests/deployments/test_cli.py

```diff
@@ -0,0 +1,69 @@
+from click.testing import CliRunner
+from mlflow.deployments import cli
+
+
+f_model_uri = 'fake_model_uri'
+f_name = 'fake_deployment_id'
+f_flavor = 'fake_flavor'
+f_target = 'fake_target'
+
+
+def test_create(patched_plugin_store):  # pylint: disable=W0613
```

Comment:
Could we add a test that uses the shorthand form of the flavor, model URI, and target URI arguments (e.g. `-f flavor -m model_uri -t target`)?